discovery-provider/src/api/v1/tracks.py

```diff
@@ -425,6 +422,13 @@ def get(self, track_id):
         if not creator_nodes or not track:
             abort_not_found(track_id, ns)
 
+        track_cid = track["track_cid"]
+        if not track_cid:
+            logger.warning(
```

Comment:
nit: if it's this bad should it be a logger.error?

Reply:
i buy it